README.rst

```diff
@@ -118,8 +113,8 @@ Meta
 .. |docs| image:: https://readthedocs.org/projects/pyobis/badge/?version=latest
    :target: http://pyobis.readthedocs.org/en/latest/?badge=latest
 
-.. |tests| image:: https://github.com/iobis/pyobis/actions/workflows/tests.yml/badge.svg
+.. |tests| image:: https://github.com/ayushanand18/pyobis/actions/workflows/tests.yml/badge.svg
    :target: https://github.com/iobis/pyobis/actions/workflows/tests.yml
 
 .. |coverage| image:: https://coveralls.io/repos/sckott/pyobis/badge.svg?branch=master&service=github
-   :target: https://coveralls.io/github/sckott/pyobis?branch=master
```

Comment:
This is pointing to the wrong repo. Need to update the url or remove the coveralls badge.

Reply:
Thanks @7yl4r for the review! I will update the URL.

Reply:
I couldn't find a way to update this a coveralls.io badge, maybe we need to register this repo at coveralls.